On windows 11, "tinygo test -target wasi  os" fails because wasi does not support cd .. yet

With a fresh windows 11 machine and today's dev (including clang 14):
```
$ ~/src/tinygo/build/tinygo test -target wasi os
Returning nil error open C:\Users\dank\AppData\Local\Temp\tinygotmp1446174477/TestFd.txt147587007: file not found
Caused by:
           1: 0x63ad - runtime.nilPanic
                           at C:\Users\dank\src\tinygo\src\runtime\panic.go:32:14
           2: 0x332f5 - (*os.File).Name
           3: 0x755c5 - os_test.TestFd
                           at C:\Users\dank\src\tinygo\src\os\file_anyos_test.go:98:21
```

Running wasmtime os.test under procmon, I noticed something funny: temporary files being opened were at the wrong location, as if wasmtime was confused by drive letters.  Building just the test by itself, then creating a writable directory C:\TMP,
then running in a CMD window, I saw that the command
```
wasmtime --dir=. --dir=.. --dir=..\.. --dir=/tmp --env=TMPDIR=/tmp os.test
```
succeeds, but the commands
```
wasmtime --dir=. --dir=.. --dir=..\.. --dir=c:/tmp --env=TMPDIR=c:/tmp os.test
wasmtime --dir=. --dir=.. --dir=..\.. --dir=\tmp --env=TMPDIR=\tmp os.test
wasmtime --dir=. --dir=.. --dir=..\.. --dir=c:\tmp --env=TMPDIR=c:\tmp os.test
```
all fail.  It seems that TMPDIR must be a unix style directory... which makes sense, since wasi is unix-ish.
And since wasmtime probably does an exact compare on the dirs to grant access, it needs the unix style directory, too.

The workaround is easy as long as TMPDIR is on the default drive...

This seems like a regression; wasn't this working before?  Maybe not; we haven't been running tinygo-test-wasm in ci on windows...

On today's dev, the test succeeds in isolation:
```
$ ~/src/tinygo/build/tinygo.exe test -target wasi -v -run 'TestFd' os
=== RUN   TestFd
--- PASS: TestFd (0.00s)
PASS
ok      os      0.144s
```
as does TestChdir:
```
$ ~/src/tinygo/build/tinygo.exe test -target wasi -v -run 'TestChdir' os
XYZZY: Run TestChdir TestChdir
=== RUN   TestChdir
--- PASS: TestChdir (0.00s)
PASS
```
But together, they explode:
```
$ ~/src/tinygo/build/tinygo.exe test -target wasi -v -run 'TestChdir|TestFd' os
Returning nil error open C:\Users\dank\AppData\Local\Temp\tinygotmp2555839111/TestFd.txt108297876: file not found
panic: runtime error: nil pointer dereference
Error: failed to run main module `C:\Users\dank\AppData\Local\Temp\tinygo32454820\main`

Caused by:
           1: 0x63ad - runtime.nilPanic
                           at C:\Users\dank\src\tinygo\src\runtime\panic.go:32:14
           2: 0x332f5 - (*os.File).Name
           3: 0x756ae - os_test.TestFd
                           at C:\Users\dank\src\tinygo\src\os\file_anyos_test.go:99:21
```
procmon shows the failed operation was a CreateFile on 
C:\Users\dank\AppData\Local\tinygo\goroot-ddc010ecf4bb48ee8dd17b18ccbda41b3242369ab05fe19cc051875e70f76a16\src\os\Users
near the beginning of TestFd.

So it seems TestChdir is misbehaving under wasi under windows somehow, breaking TestFd.

Adding a print of the current directory at the start of TestFd shows:
```
XYZZY: GetWd returns /_os_test_TestChDir/.. 
```

Evidently chdir under wasi is vestigal, and doesn't know enough to change to the parent directory when you change to ".."

Why this only causes test failures under windows is a mystery.

--- a/.github/workflows/windows.yml
+++ b/.github/workflows/windows.yml
@@ -71,6 +71,9 @@ jobs:
       - name: Build wasi-libc
         if: steps.cache-wasi-libc.outputs.cache-hit != 'true'
         run: make wasi-libc
+      - name: Install wasmtime
+        run: |
+          scoop install wasmtime
       - name: Test TinyGo
         shell: bash
         run: make test GOTESTFLAGS="-v -short"
@@ -96,3 +99,5 @@ jobs:
         run: make smoketest TINYGO=$(PWD)/build/tinygo AVR=0 XTENSA=0
       - name: Test stdlib packages
         run: make tinygo-test
+      - name: Test stdlib packages on wasi
+        run: make tinygo-test-wasi-fast

--- a/Makefile
+++ b/Makefile
@@ -221,7 +221,6 @@ TEST_PACKAGES_SLOW = \
 
 # Standard library packages that pass tests quickly on darwin, linux, wasi, and windows
 TEST_PACKAGES_FAST = \
-	compress/lzw \
 	compress/zlib \
 	container/heap \
 	container/list \
@@ -269,17 +268,22 @@ TEST_PACKAGES_FAST = \
 # io/fs requires os.ReadDir, which is not yet supported on windows or wasi
 # testing/fstest requires os.ReadDir, which is not yet supported on windows or wasi
 # compress/flate fails windows go 1.18, https://github.com/tinygo-org/tinygo/issues/2762
+# compress/lzw fails windows go 1.18 wasi, https://github.com/tinygo-org/tinygo/issues/2762
 
 # Additional standard library packages that pass tests on individual platforms
 TEST_PACKAGES_LINUX := \
 	archive/zip \
 	compress/flate \
+	compress/lzw \
 	debug/dwarf \
 	debug/plan9obj \
 	io/fs \
 	testing/fstest
 
 TEST_PACKAGES_DARWIN := $(TEST_PACKAGES_LINUX)
+
+TEST_PACKAGES_WINDOWS := \
+	compress/lzw
 
 # Report platforms on which each standard library package is known to pass tests
 jointmp := $(shell echo /tmp/join.$$$$)
@@ -299,7 +303,7 @@ ifeq ($(shell uname),Linux)
 TEST_PACKAGES_HOST := $(TEST_PACKAGES_FAST) $(TEST_PACKAGES_LINUX)
 endif
 ifeq ($(OS),Windows_NT)
-TEST_PACKAGES_HOST := $(TEST_PACKAGES_FAST)
+TEST_PACKAGES_HOST := $(TEST_PACKAGES_FAST) $(TEST_PACKAGES_WINDOWS)
 endif
 
 # Test known-working standard library packages.

--- a/src/os/file_anyos_test.go
+++ b/src/os/file_anyos_test.go
@@ -31,9 +31,14 @@ func TestTempDir(t *testing.T) {
 
 func TestChdir(t *testing.T) {
 	// create and cd into a new directory
+	oldDir, err := Getwd()
+	if err != nil {
+		t.Errorf("Getwd() returned %v", err)
+		return
+	}
 	dir := "_os_test_TestChDir"
 	Remove(dir)
-	err := Mkdir(dir, 0755)
+	err = Mkdir(dir, 0755)
 	defer Remove(dir) // even though not quite sure which directory it will execute in
 	if err != nil {
 		t.Errorf("Mkdir(%s, 0755) returned %v", dir, err)
@@ -55,7 +60,9 @@ func TestChdir(t *testing.T) {
 		t.Errorf("Close %s: %s", file, err)
 	}
 	// cd back to original directory
-	err = Chdir("..")
+	// TODO: emulate "cd .." in wasi-libc better?
+	//err = Chdir("..")
+	err = Chdir(oldDir)
 	if err != nil {
 		t.Errorf("Chdir ..: %s", err)
 	}

